pkg/pb/metainfo.proto

```diff
@@ -28,6 +28,14 @@ service Metainfo {
     rpc BeginDeleteObject(ObjectBeginDeleteRequest) returns (ObjectBeginDeleteResponse);
     rpc FinishDeleteObject(ObjectFinishDeleteRequest) returns (ObjectFinishDeleteResponse);
 
+    rpc BeginSegment(SegmentBeginRequest) returns (SegmentBeginResponse);
+    rpc CommitSegment(SegmentCommitRequest) returns (SegmentCommitResponse);
+    rpc MakeInlineSegment(SegmentMakeInlineRequest) returns (SegmentMakeInlineResponse);
```

Comment:
**Just a thought.** Why do we need to separately tell satellite that store this segment ( which apparently happened to be a small size) as inline. can we(uplink) just encode like it does with other pieces(bigger than inline size) and let satellite decide based on the size of the piece, to store it as inline or remote. In short eliminate the uplink making the decision and telling satellite to store it has inline or remote.

Reply:
the inline segment size is an uplink decision, can be larger than our current default but price for it will be much higher than remote stuff